### PR TITLE
prevent site rendering in iframe (task #1735)

### PR DESCRIPTION
**Pull request by georgeconstantinou**

### CODE REVIEW

File: src/Controller/AppController.php [hunk 1]
```diff
@@ -54,6 +54,11 @@ public function initialize()
      */
     public function beforeFilter(Event $event)
     {
+        /*
+        Prevent page rendering in iframe
+         */
+        $this->response->header('X-Frame-Options', 'DENY');
```
Comment (mamchenkov) [inline src/Controller/AppController.php]: @georgeconstantinou for our case, it's better to have a .env.example variable set to DENY, rather then hardcode it, as there might be cases where the application will need to be inside an iframe.

**Comment by georgeconstantinou** [comment]
@mamchenkov please review again.

**Comment by flangofas** [comment]
:+1:

**Comment by leonidsl** [comment]
:+1:

**Pull request merged**